Sources/EpoxyCore/SwiftUI/EpoxySwiftUIHostingView.swift

```diff
@@ -308,23 +314,43 @@ extension UIResponder {
   }
 }
 
+// MARK: - EpoxyHostingContent
+
+/// The object that is used to communicate changes in the root view to the
+/// `EpoxySwiftUIHostingController`.
```

Comment:
Can we add some discussion about how communicating the content updates via this observable object fixes the issue with propagating transactions that occurs with the `UIHostingController.rootView =` approach?

Reply:
Added to line 134